backend/tournesol/resources/export_readme.txt

```diff
@@ -18,43 +18,116 @@ List of files
 comparisons.csv
 ---------------
 
-Type: algorithm input.
-
 This file contains all the public comparisons made by users. A comparison is
 represented by a rating given by a user for a specific criteria and a pair of
 videos.
 
+List of columns:
+
+- public_username:
+  The username of the user who submitted the comparison.
+- video_a:
+  One of the two video being compared. If the score is negative, it indicates
+  that the user prefers video_a against video_b.
```

Comment:
I like having a blank line between the columns, I think it will help the reading. What do you think?

Reply:
Done